frontend/src/metabase/query_builder/selectors.js

```diff
@@ -594,19 +594,22 @@ export const getIsObjectDetail = createSelector(
   (mode, isZoomingSingleRow) => isZoomingSingleRow || mode?.name() === "object",
 );
 
+export function isQuestionDirty(question, originalQuestion) {
```

Comment:
should we move these functions to a different folder? Maybe we can have some question functions and hooks (like the createQuestion and updateQuestion hooks that I made a few days ago) in a common folder that the main and SDK apps can use